### PR TITLE
Fix column resizing misalignment

### PR DESCRIPTION
**Pull request by budnix**
### Context

Fixed by correcting column index, translation from renderable to visual index. Added missing tests for the _HiddenColumns_ and _HiddenRows_ plugin.

### How has this been tested?

Added E2E tests.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1.  #6867
2.
3.

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: src/plugins/hiddenColumns/test/plugins/manualColumnResize.e2e.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          columns: [1],
+          indicators: true
+        },
+        manualColumnResize: true,
+      });
+
+      expect(colWidth(spec().$container, 1)).toBe(65); // 50 + 15 (indicator)
+
+      // Resize renderable column index `1` (within visual index term the index at 1 is hidden).
+      resizeColumn(1, 100);
```
Comment (wszymanski) [inline src/plugins/hiddenColumns/test/plugins/manualColumnResize.e2e.js:30]: Could you change also the argument description in the `resizeColumn` function from the `common.js` file?

File: src/plugins/manualColumnResize/manualColumnResize.js [hunk 2]
```diff
@@ -228,7 +227,7 @@ class ManualColumnResize extends BasePlugin {
         relativeHeaderPosition = parentOverlay.getRelativeCellPosition(topMostHeader, cellCoords.row, cellCoords.col);
       }
 
-      this.currentCol = col;
+      this.currentCol = this.hot.columnIndexMapper.getVisualFromRenderableIndex(col);
```
Comment (wszymanski) [inline src/plugins/manualColumnResize/manualColumnResize.js:230]: One line which makes a difference! 🙂

File: src/plugins/manualColumnResize/manualColumnResize.js [hunk 3]
```diff
@@ -343,8 +342,8 @@ class ManualColumnResize extends BasePlugin {
       if (element.tagName === 'TH') {
         return element;
       }
-      return this.getTHFromTargetElement(element.parentNode);
 
+      return this.getTHFromTargetElement(element.parentNode);
```
Comment (wszymanski) [inline src/plugins/manualColumnResize/manualColumnResize.js:346]: Nice to see API call, although the function looks stange :P

File: src/plugins/hiddenColumns/test/plugins/manualColumnResize.e2e.js (hunk 1 above)
Comment (budnix) [reply src/plugins/hiddenColumns/test/plugins/manualColumnResize.e2e.js:30]: Fixed within https://github.com/handsontable/handsontable/pull/6940/commits/dd3a20de608de1a3322c34d078f979494c1c794d.

**Pull request merged**